### PR TITLE
Create AWS EMR Serverless Fabric page

### PR DESCRIPTION
**Pull request by alexanderahn**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208106093127607

### CODE REVIEW

File: docs/Spark/fabrics/emr-fabric-serverless.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Enter your AWS credentials under **Access Key** and **Secret Key**. Choose the **Region** that your EMR Serverless cluster is running in.
+
+**Click** on **Fetch environments**.
+<img src={require('./img/emr-aws-cred.png').default} alt="EMR cred" width="75%" />
+
+**Select** your EMR Serverless cluster that you would like to connect to under **Spark Environment**.
+
+<img src={require('./img/emr-serverless-dev.png').default} alt="EMR select" width="80%" />
+
+Most of the fields should be automatically populated after selecting a EMR Serverless cluster. Enter the S3 path that points to the location where you would like your logs to persist.
```
Comment (ashvjit-prophecy) [inline docs/Spark/fabrics/emr-fabric-serverless.md:66]: *logs and pipeline artifacts

Comment (alexanderahn) [reply docs/Spark/fabrics/emr-fabric-serverless.md:66]: Done.

File: docs/Spark/fabrics/emr-fabric-serverless.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Select your EMR Serverless cluster that you would like to connect to under **Spark Environment**. The EMR cluster listing also shows Serverless clusters. From the list of active clusters, you can identify your EMR Serverless cluster by finding `emr-serverless-services` within the Livy URL.
+
+<img src={require('./img/emr-serverless-dev-select.png').default} alt="EMR Serverless select" width="80%" />
+
+Most of the fields should be automatically populated after selecting a EMR Serverless cluster.
+
+Optional: You can enable Lake Formation by clicking the toggle.
+
+Select your Authentication type. We suggest choosing `AWS Sig V4`.
+<img src={require('./img/emr-serverless-auth-type.png').default} alt="EMR Serverless Authentication type" width="50%" />
```
Comment (ashvjit-prophecy) [inline docs/Spark/fabrics/emr-fabric-serverless.md:71]: For EMR serverless, this is the only auth that will work. So, can we use a stronger word than "suggest"?

File: docs/Spark/fabrics/emr-fabric-serverless.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+Select your Runtime Role. You will only see roles that you have access to.
+<img src={require('./img/emr-serverless-runtime-role.png').default} alt="EMR Serverless Runtime Role" width="75%" />
```
Comment (ashvjit-prophecy) [inline docs/Spark/fabrics/emr-fabric-serverless.md:74]: Something like "Select your Runtime Role. This role should have enough permissions to use the selected Serverless App"

File: docs/Spark/fabrics/emr-fabric-serverless.md (hunk 2 above)
Comment (alexanderahn) [reply docs/Spark/fabrics/emr-fabric-serverless.md:71]: Done.

File: docs/Spark/fabrics/emr-fabric-serverless.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+Click on **Fetch environments**.
+<img src={require('./img/emr-aws-cred.png').default} alt="EMR Serverless cred" width="75%" />
+
+Select your EMR Serverless cluster that you would like to connect to under **Spark Environment**. The EMR cluster listing also shows Serverless clusters. From the list of active clusters, you can identify your EMR Serverless cluster by finding `emr-serverless-services` within the Livy URL.
+
+<img src={require('./img/emr-serverless-dev-select.png').default} alt="EMR Serverless select" width="80%" />
+
+Most of the fields should be automatically populated after selecting a EMR Serverless cluster.
+
+Optional: You can enable Lake Formation by clicking the toggle.
```
[the remark this answers is not in the document]
Comment (alexanderahn) [reply docs/Spark/fabrics/emr-fabric-serverless.md:68]: Done.

File: docs/Spark/fabrics/emr-fabric-serverless.md (hunk 3 above)
Comment (alexanderahn) [reply docs/Spark/fabrics/emr-fabric-serverless.md:74]: Done.

Comment (alexanderahn) [reply docs/Spark/fabrics/emr-fabric-serverless.md:74]: Thank you for the comments :)

**Review by ashvjit-prophecy — APPROVED** [review]
LGTM

**Pull request merged**